src/index.html

```diff
@@ -13,6 +13,40 @@
     />
```

Comment:
According to the checklist, you should link the final CSS file, not the SCSS source file. Make sure to compile your SCSS to CSS and link to the CSS file instead.